addons/hr_homeworking/static/src/calendar/common/calendar_common_renderer.js

```diff
@@ -98,8 +98,8 @@ patch(AttendeeCalendarCommonRenderer.prototype, {
             if (!button || !line)
                 return;
             info.homework = true;
-            button.onclick = () =>this.onDateClick(info)
-            line.onclick = () =>this.onDateClick(info)
+            button.onclick = () => this.props.createRecord(this.fcEventToRecord(info));
+            line.onclick = () => this.props.createRecord(this.fcEventToRecord(info));
```

Comment:
we should rather modify `onDateClick`, or remove it, if it's not needed

Reply:
I discussed this with the JS framework team, and as `onDateClick` is part of the fullcalendar API, they rather not have any manual calls to it, especially when the `info` object doesn't conform to the API.

The `onDateClick` override also cannot be removed, as it enables setting locations from month overview, while `onDayRender` enables it from day & week overviews.